### PR TITLE
Fixes and new report command

### PR DESCRIPTION
**Pull request by toabctl**

### CODE REVIEW

**Comment by toabctl** [comment]
Tested with:
1) generating the quickstart_entries.json file with:
```
python -m aws_marketplace_ubuntu_scraper quicklaunch --iam-account-id=XXX --iam-username=XXX --iam-password="XXX" --no-headless
```
2) generating the report with:
```
python -m aws_marketplace_ubuntu_scraper report
```

that results in:
```
$ python -m aws_marketplace_ubuntu_scraper report
Checking region af-south-1 ...
Checking region ap-east-1 ...
Checking region ap-northeast-1 ...
Checking region ap-northeast-2 ...
Checking region ap-northeast-3 ...
Checking region ap-south-1 ...
Checking region ap-southeast-1 ...
Checking region ap-southeast-2 ...
Checking region ap-southeast-3 ...
Checking region ca-central-1 ...
Checking region eu-central-1 ...
Checking region eu-north-1 ...
Checking region eu-south-1 ...
Checking region eu-west-1 ...
Checking region eu-west-2 ...
Checking region eu-west-3 ...
Checking region me-south-1 ...
Checking region sa-east-1 ...
Checking region us-east-1 ...
Checking region us-east-2 ...
Checking region us-west-1 ...
Checking region us-west-2 ...
+----------------+---------+-------+----------+-----------------------+-----------------------+--------------+
|     Region     | Release |  Arch | Position |     Quickstart AMI    |      Streams AMI      | Needs update |
+----------------+---------+-------+----------+-----------------------+-----------------------+--------------+
|   us-west-2    |  20.04  | arm64 |    8     | ami-078278691222aee06 | ami-078278691222aee06 |    False     |
|                |         |       |          |                       |                       |              |
|   us-west-2    |  20.04  | amd64 |    8     | ami-0892d3c7ee96c0bf7 | ami-0892d3c7ee96c0bf7 |    False     |
|                |         |       |          |                       |                       |              |
|   us-west-2    |  18.04  | arm64 |    9     | ami-08d20da2261ebe2ec | ami-08d20da2261ebe2ec |    False     |
|                |         |       |          |                       |                       |              |
|   us-west-2    |  18.04  | amd64 |    9     | ami-074251216af698218 | ami-074251216af698218 |    False     |
|                |         |       |          |                       |                       |              |
|   us-west-2    |  16.04  | arm64 |    41    | ami-08305dd8ab642ad8c | ami-08305dd8ab642ad8c |    False     |
|                |         |       |          |                       |                       |              |
|   us-west-2    |  16.04  | amd64 |    41    | ami-0688ba7eeeeefe3cd | ami-0688ba7eeeeefe3cd |    False     |
|                |         |       |          |                       |                       |              |
|   us-west-1    |  20.04  | arm64 |    5     | ami-039da1612f6f41296 | ami-039da1612f6f41296 |    False     |
|                |         |       |          |                       |                       |              |
|   us-west-1    |  20.04  | amd64 |    5     | ami-01f87c43e618bf8f0 | ami-01f87c43e618bf8f0 |    False     |
|                |         |       |          |                       |                       |              |
|   us-west-1    |  18.04  | arm64 |    6     | ami-0f561e14031ed52d3 | ami-0f561e14031ed52d3 |    False     |
|                |         |       |          |                       |                       |              |
|   us-west-1    |  18.04  | amd64 |    6     | ami-009726b835c24a3aa | ami-009726b835c24a3aa |    False     |
|                |         |       |          |                       |                       |              |
|   us-west-1    |  16.04  | arm64 |    36    | ami-06d79d5316d395362 | ami-06d79d5316d395362 |    False     |
|                |         |       |          |                       |                       |              |
|   us-west-1    |  16.04  | amd64 |    36    | ami-0454207e5367abf01 | ami-0454207e5367abf01 |    False     |
|                |         |       |          |                       |                       |              |
|   us-east-2    |  20.04  | arm64 |    8     | ami-02af65b2d1ebdfafc | ami-02af65b2d1ebdfafc |    False     |
|                |         |       |          |                       |                       |              |
|   us-east-2    |  20.04  | amd64 |    8     | ami-0fb653ca2d3203ac1 | ami-0fb653ca2d3203ac1 |    False     |
|                |         |       |          |                       |                       |              |
|   us-east-2    |  18.04  | arm64 |    28    | ami-023d39cbc16614424 | ami-023d39cbc16614424 |    False     |
|                |         |       |          |                       |                       |              |
|   us-east-2    |  18.04  | amd64 |    28    | ami-03a0c45ebc70f98ea | ami-03a0c45ebc70f98ea |    False     |
|                |         |       |          |                       |                       |              |
|   us-east-2    |  16.04  | arm64 |    40    | ami-0b76100074ce446d4 | ami-0b76100074ce446d4 |    False     |
|                |         |       |          |                       |                       |              |
|   us-east-2    |  16.04  | amd64 |    40    | ami-05803413c51f242b7 | ami-05803413c51f242b7 |    False     |
|                |         |       |          |                       |                       |              |
|   us-east-1    |  20.04  | arm64 |    8     | ami-0b49a4a6e8e22fa16 | ami-0b49a4a6e8e22fa16 |    False     |
|                |         |       |          |                       |                       |              |
|   us-east-1    |  20.04  | amd64 |    8     | ami-04505e74c0741db8d | ami-04505e74c0741db8d |    False     |
|                |         |       |          |                       |                       |              |
|   us-east-1    |  18.04  | arm64 |    9     | ami-0a940cb939351ccca | ami-0a940cb939351ccca |    False     |
|                |         |       |          |                       |                       |              |
|   us-east-1    |  18.04  | amd64 |    9     | ami-0e472ba40eb589f49 | ami-0e472ba40eb589f49 |    False     |
|                |         |       |          |                       |                       |              |
|   us-east-1    |  16.04  | arm64 |    41    | ami-04b3c39a8a1c62b76 | ami-04b3c39a8a1c62b76 |    False     |
|                |         |       |          |                       |                       |              |
|   us-east-1    |  16.04  | amd64 |    41    | ami-0b0ea68c435eb488d | ami-0b0ea68c435eb488d |    False     |
|                |         |       |          |                       |                       |              |
|   sa-east-1    |  20.04  | arm64 |    5     | ami-0e9fbe8acb00ab545 | ami-0e9fbe8acb00ab545 |    False     |
|                |         |       |          |                       |                       |              |
|   sa-east-1    |  20.04  | amd64 |    5     | ami-090006f29ecb2d79a | ami-090006f29ecb2d79a |    False     |
|                |         |       |          |                       |                       |              |
|   sa-east-1    |  18.04  | arm64 |    9     | ami-0399e2debd94407c8 | ami-0399e2debd94407c8 |    False     |
|                |         |       |          |                       |                       |              |
|   sa-east-1    |  18.04  | amd64 |    9     | ami-07b5a89195c6932c8 | ami-07b5a89195c6932c8 |    False     |
|                |         |       |          |                       |                       |              |
|   sa-east-1    |  16.04  | arm64 |    38    | ami-08bf1084c0b9a3361 | ami-08bf1084c0b9a3361 |    False     |
|                |         |       |          |                       |                       |              |
|   sa-east-1    |  16.04  | amd64 |    38    | ami-0a729bdc1acf7528b | ami-0a729bdc1acf7528b |    False     |
|                |         |       |          |                       |                       |              |
|   me-south-1   |  20.04  | arm64 |    4     | ami-0e5334860d39cfacd | ami-0cf05bd11a7fc76d8 |    False     |
|                |         |       |          |                       |                       |              |
|   me-south-1   |  20.04  | amd64 |    4     | ami-0b4946d7420c44be4 | ami-0f7b49dc4bc2b14cf |    False     |
|                |         |       |          |                       |                       |              |
|   me-south-1   |  18.04  | arm64 |    5     | ami-0849dc855dcd90c30 | ami-08b01b80e82f8740a |    False     |
|                |         |       |          |                       |                       |              |
|   me-south-1   |  18.04  | amd64 |    5     | ami-0ef669c57b73af73b | ami-0ee4660316d1f73ba |    False     |
|                |         |       |          |                       |                       |              |
|   me-south-1   |  16.04  | arm64 |    31    | ami-0cebc6b71fba48617 | ami-0cebc6b71fba48617 |    False     |
|                |         |       |          |                       |                       |              |
|   me-south-1   |  16.04  | amd64 |    31    | ami-0c41538a47f4b7d47 | ami-0c41538a47f4b7d47 |    False     |
|                |         |       |          |                       |                       |              |
|   eu-west-3    |  20.04  | arm64 |    4     | ami-0df2138f920780ecc | ami-0df2138f920780ecc |    False     |
|                |         |       |          |                       |                       |              |
|   eu-west-3    |  20.04  | amd64 |    4     | ami-0c6ebbd55ab05f070 | ami-0c6ebbd55ab05f070 |    False     |
|                |         |       |          |                       |                       |              |
|   eu-west-3    |  18.04  | arm64 |    5     | ami-0815a6f5c055c7ae8 | ami-0815a6f5c055c7ae8 |    False     |
|                |         |       |          |                       |                       |              |
|   eu-west-3    |  18.04  | amd64 |    5     | ami-06ad2ef8cd7012912 | ami-06ad2ef8cd7012912 |    False     |
|                |         |       |          |                       |                       |              |
|   eu-west-3    |  16.04  | arm64 |    36    | ami-0a5ee67e16e2e3530 | ami-0a5ee67e16e2e3530 |    False     |
|                |         |       |          |                       |                       |              |
|   eu-west-3    |  16.04  | amd64 |    36    | ami-052f10f1c45aa2155 | ami-052f10f1c45aa2155 |    False     |
|                |         |       |          |                       |                       |              |
|   eu-west-2    |  20.04  | arm64 |    8     | ami-0100b57d679daec93 | ami-0100b57d679daec93 |    False     |
|                |         |       |          |                       |                       |              |
|   eu-west-2    |  20.04  | amd64 |    8     | ami-0015a39e4b7c0966f | ami-0015a39e4b7c0966f |    False     |
|                |         |       |          |                       |                       |              |
|   eu-west-2    |  18.04  | arm64 |    30    | ami-05d9b862442ea07db | ami-05d9b862442ea07db |    False     |
|                |         |       |          |                       |                       |              |
|   eu-west-2    |  18.04  | amd64 |    30    | ami-0f9124f7452cdb2a6 | ami-0f9124f7452cdb2a6 |    False     |
|                |         |       |          |                       |                       |              |
|   eu-west-2    |  16.04  | arm64 |    40    | ami-01c0540df77ccaa68 | ami-01c0540df77ccaa68 |    False     |
|                |         |       |          |                       |                       |              |
|   eu-west-2    |  16.04  | amd64 |    40    | ami-09a2a0f7d2db8baca | ami-09a2a0f7d2db8baca |    False     |
|                |         |       |          |                       |                       |              |
|   eu-west-1    |  20.04  | arm64 |    8     | ami-0c0a42948ea1b4f44 | ami-0c0a42948ea1b4f44 |    False     |
|                |         |       |          |                       |                       |              |
|   eu-west-1    |  20.04  | amd64 |    8     | ami-08ca3fed11864d6bb | ami-08ca3fed11864d6bb |    False     |
|                |         |       |          |                       |                       |              |
|   eu-west-1    |  18.04  | arm64 |    9     | ami-01ddc0aefcdbc53e7 | ami-01ddc0aefcdbc53e7 |    False     |
|                |         |       |          |                       |                       |              |
|   eu-west-1    |  18.04  | amd64 |    9     | ami-07d8796a2b0f8d29c | ami-07d8796a2b0f8d29c |    False     |
|                |         |       |          |                       |                       |              |
|   eu-west-1    |  16.04  | arm64 |    40    | ami-0e8225827581c983a | ami-0e8225827581c983a |    False     |
|                |         |       |          |                       |                       |              |
|   eu-west-1    |  16.04  | amd64 |    40    | ami-0f29c8402f8cce65c | ami-0f29c8402f8cce65c |    False     |
|                |         |       |          |                       |                       |              |
|   eu-south-1   |  20.04  | arm64 |    4     | ami-008c11e98baa0896c | ami-065baefce8b761cc5 |    False     |
|                |         |       |          |                       |                       |              |
|   eu-south-1   |  20.04  | amd64 |    4     | ami-0f8ce9c417115413d | ami-0d49110b200671cd8 |    False     |
|                |         |       |          |                       |                       |              |
|   eu-south-1   |  18.04  | arm64 |    5     | ami-03d503f6eef41cde7 | ami-0f81729e06cbc6cc2 |    False     |
|                |         |       |          |                       |                       |              |
|   eu-south-1   |  18.04  | amd64 |    5     | ami-09f165dd6bd167be5 | ami-01cbeefbb66612ca6 |    False     |
|                |         |       |          |                       |                       |              |
|   eu-south-1   |  16.04  | arm64 |    32    | ami-0388cd84798fdb1a9 | ami-0388cd84798fdb1a9 |    False     |
|                |         |       |          |                       |                       |              |
|   eu-south-1   |  16.04  | amd64 |    32    | ami-027f7881d2f6725e1 | ami-027f7881d2f6725e1 |    False     |
|                |         |       |          |                       |                       |              |
|   eu-north-1   |  20.04  | arm64 |    7     | ami-0b50645ac1261267b | ami-0b50645ac1261267b |    False     |
|                |         |       |          |                       |                       |              |
|   eu-north-1   |  20.04  | amd64 |    7     | ami-092cce4a19b438926 | ami-092cce4a19b438926 |    False     |
|                |         |       |          |                       |                       |              |
|   eu-north-1   |  18.04  | arm64 |    8     | ami-02f66f80e3779c743 | ami-02f66f80e3779c743 |    False     |
|                |         |       |          |                       |                       |              |
|   eu-north-1   |  18.04  | amd64 |    8     | ami-05760f62e0b3eab56 | ami-05760f62e0b3eab56 |    False     |
|                |         |       |          |                       |                       |              |
|   eu-north-1   |  16.04  | arm64 |    36    | ami-083ac04a8442d4b7f | ami-083ac04a8442d4b7f |    False     |
|                |         |       |          |                       |                       |              |
|   eu-north-1   |  16.04  | amd64 |    36    | ami-000e50175c5f86214 | ami-000e50175c5f86214 |    False     |
|                |         |       |          |                       |                       |              |
|  eu-central-1  |  20.04  | arm64 |    8     | ami-0b168c89474ef4301 | ami-0b168c89474ef4301 |    False     |
|                |         |       |          |                       |                       |              |
|  eu-central-1  |  20.04  | amd64 |    8     | ami-0d527b8c289b4af7f | ami-0d527b8c289b4af7f |    False     |
|                |         |       |          |                       |                       |              |
|  eu-central-1  |  18.04  | arm64 |    25    | ami-0903fdada63611430 | ami-0903fdada63611430 |    False     |
|                |         |       |          |                       |                       |              |
|  eu-central-1  |  18.04  | amd64 |    25    | ami-042ad9eec03638628 | ami-042ad9eec03638628 |    False     |
|                |         |       |          |                       |                       |              |
|  eu-central-1  |  16.04  | arm64 |    40    | ami-074bcbbba36789937 | ami-074bcbbba36789937 |    False     |
|                |         |       |          |                       |                       |              |
|  eu-central-1  |  16.04  | amd64 |    40    | ami-09042b2f6d07d164a | ami-09042b2f6d07d164a |    False     |
|                |         |       |          |                       |                       |              |
|  ca-central-1  |  20.04  | arm64 |    5     | ami-04e842c4cdd82c62e | ami-04e842c4cdd82c62e |    False     |
|                |         |       |          |                       |                       |              |
|  ca-central-1  |  20.04  | amd64 |    5     | ami-0aee2d0182c9054ac | ami-0aee2d0182c9054ac |    False     |
|                |         |       |          |                       |                       |              |
|  ca-central-1  |  18.04  | arm64 |    27    | ami-047c4c0a140c45587 | ami-047c4c0a140c45587 |    False     |
|                |         |       |          |                       |                       |              |
|  ca-central-1  |  18.04  | amd64 |    27    | ami-04a6c66cacdef1be5 | ami-04a6c66cacdef1be5 |    False     |
|                |         |       |          |                       |                       |              |
|  ca-central-1  |  16.04  | arm64 |    37    | ami-03fc4d6b3ea7e9081 | ami-03fc4d6b3ea7e9081 |    False     |
|                |         |       |          |                       |                       |              |
|  ca-central-1  |  16.04  | amd64 |    37    | ami-03bcd79f25ca6b127 | ami-03bcd79f25ca6b127 |    False     |
|                |         |       |          |                       |                       |              |
| ap-southeast-2 |  20.04  | arm64 |    8     | ami-07ef508d01f533f5f | ami-07ef508d01f533f5f |    False     |
|                |         |       |          |                       |                       |              |
| ap-southeast-2 |  20.04  | amd64 |    8     | ami-0b7dcd6e6fd797935 | ami-0b7dcd6e6fd797935 |    False     |
|                |         |       |          |                       |                       |              |
| ap-southeast-2 |  18.04  | arm64 |    10    | ami-06766ee000a27e80b | ami-06766ee000a27e80b |    False     |
|                |         |       |          |                       |                       |              |
| ap-southeast-2 |  18.04  | amd64 |    10    | ami-05654370f5b5eb0b0 | ami-05654370f5b5eb0b0 |    False     |
|                |         |       |          |                       |                       |              |
| ap-southeast-2 |  16.04  | arm64 |    40    | ami-09fe2be72ca884906 | ami-09fe2be72ca884906 |    False     |
|                |         |       |          |                       |                       |              |
| ap-southeast-2 |  16.04  | amd64 |    40    | ami-0672b175139a0f8f4 | ami-0672b175139a0f8f4 |    False     |
|                |         |       |          |                       |                       |              |
| ap-southeast-1 |  20.04  | arm64 |    8     | ami-03d20fc12884694ee | ami-03d20fc12884694ee |    False     |
|                |         |       |          |                       |                       |              |
| ap-southeast-1 |  20.04  | amd64 |    8     | ami-055d15d9cfddf7bd3 | ami-055d15d9cfddf7bd3 |    False     |
|                |         |       |          |                       |                       |              |
| ap-southeast-1 |  18.04  | arm64 |    9     | ami-035d952af1e83a0d9 | ami-035d952af1e83a0d9 |    False     |
|                |         |       |          |                       |                       |              |
| ap-southeast-1 |  18.04  | amd64 |    9     | ami-07315f74f3fa6a5a3 | ami-07315f74f3fa6a5a3 |    False     |
|                |         |       |          |                       |                       |              |
| ap-southeast-1 |  16.04  | arm64 |    40    | ami-03b6c12f592f54cf3 | ami-03b6c12f592f54cf3 |    False     |
|                |         |       |          |                       |                       |              |
| ap-southeast-1 |  16.04  | amd64 |    40    | ami-0f74c08b8b5effa56 | ami-0f74c08b8b5effa56 |    False     |
|                |         |       |          |                       |                       |              |
|   ap-south-1   |  20.04  | arm64 |    8     | ami-0491e5015eb6e7a9b | ami-0491e5015eb6e7a9b |    False     |
|                |         |       |          |                       |                       |              |
|   ap-south-1   |  20.04  | amd64 |    8     | ami-0851b76e8b1bce90b | ami-0851b76e8b1bce90b |    False     |
|                |         |       |          |                       |                       |              |
|   ap-south-1   |  18.04  | arm64 |    24    | ami-0f3b400f4ed1c4eda | ami-0f3b400f4ed1c4eda |    False     |
|                |         |       |          |                       |                       |              |
|   ap-south-1   |  18.04  | amd64 |    24    | ami-08ee6644906ff4d6c | ami-08ee6644906ff4d6c |    False     |
|                |         |       |          |                       |                       |              |
|   ap-south-1   |  16.04  | arm64 |    40    | ami-0ae12517fe595ce17 | ami-0ae12517fe595ce17 |    False     |
|                |         |       |          |                       |                       |              |
|   ap-south-1   |  16.04  | amd64 |    40    | ami-0f2e255ec956ade7f | ami-0f2e255ec956ade7f |    False     |
|                |         |       |          |                       |                       |              |
| ap-northeast-3 |  20.04  | arm64 |    5     | ami-0032f319e7eb9706a | ami-0032f319e7eb9706a |    False     |
|                |         |       |          |                       |                       |              |
| ap-northeast-3 |  20.04  | amd64 |    5     | ami-096c4b6e0792d8c16 | ami-096c4b6e0792d8c16 |    False     |
|                |         |       |          |                       |                       |              |
| ap-northeast-3 |  18.04  | arm64 |    6     | ami-04325faa4ff3762cf | ami-04325faa4ff3762cf |    False     |
|                |         |       |          |                       |                       |              |
| ap-northeast-3 |  18.04  | amd64 |    6     | ami-04e1b6c20fd8a3fef | ami-04e1b6c20fd8a3fef |    False     |
|                |         |       |          |                       |                       |              |
| ap-northeast-3 |  16.04  | arm64 |    31    | ami-0ab5b86f9b84122c8 | ami-0ab5b86f9b84122c8 |    False     |
|                |         |       |          |                       |                       |              |
| ap-northeast-3 |  16.04  | amd64 |    31    | ami-00f5d213b513f1b07 | ami-00f5d213b513f1b07 |    False     |
|                |         |       |          |                       |                       |              |
| ap-northeast-2 |  20.04  | arm64 |    6     | ami-03665e633d23b2df6 | ami-03665e633d23b2df6 |    False     |
|                |         |       |          |                       |                       |              |
| ap-northeast-2 |  20.04  | amd64 |    6     | ami-0454bb2fefc7de534 | ami-0454bb2fefc7de534 |    False     |
|                |         |       |          |                       |                       |              |
| ap-northeast-2 |  18.04  | arm64 |    7     | ami-0bb9d4fcf7aa7336d | ami-0bb9d4fcf7aa7336d |    False     |
|                |         |       |          |                       |                       |              |
| ap-northeast-2 |  18.04  | amd64 |    7     | ami-0ed11f3863410c386 | ami-0ed11f3863410c386 |    False     |
|                |         |       |          |                       |                       |              |
| ap-northeast-2 |  16.04  | arm64 |    38    | ami-0b37e6563ef42a0eb | ami-0b37e6563ef42a0eb |    False     |
|                |         |       |          |                       |                       |              |
| ap-northeast-2 |  16.04  | amd64 |    38    | ami-0dd97ebb907cf9366 | ami-0dd97ebb907cf9366 |    False     |
|                |         |       |          |                       |                       |              |
| ap-northeast-1 |  20.04  | arm64 |    9     | ami-0fe0d4d94786add8e | ami-0fe0d4d94786add8e |    False     |
|                |         |       |          |                       |                       |              |
| ap-northeast-1 |  20.04  | amd64 |    9     | ami-088da9557aae42f39 | ami-088da9557aae42f39 |    False     |
|                |         |       |          |                       |                       |              |
| ap-northeast-1 |  18.04  | arm64 |    10    | ami-026f11de47d216856 | ami-026f11de47d216856 |    False     |
|                |         |       |          |                       |                       |              |
| ap-northeast-1 |  18.04  | amd64 |    10    | ami-00bc9b7f0e98dc134 | ami-00bc9b7f0e98dc134 |    False     |
|                |         |       |          |                       |                       |              |
| ap-northeast-1 |  16.04  | arm64 |    40    | ami-06602ea953ba135a9 | ami-06602ea953ba135a9 |    False     |
|                |         |       |          |                       |                       |              |
| ap-northeast-1 |  16.04  | amd64 |    40    | ami-0822295a729d2a28e | ami-0822295a729d2a28e |    False     |
|                |         |       |          |                       |                       |              |
|   ap-east-1    |  20.04  | arm64 |    5     | ami-0a8d92dc5d81fa22b | ami-0a8d92dc5d81fa22b |    False     |
|                |         |       |          |                       |                       |              |
|   ap-east-1    |  20.04  | amd64 |    5     | ami-0b981d9ee99b28eba | ami-0b981d9ee99b28eba |    False     |
|                |         |       |          |                       |                       |              |
|   ap-east-1    |  18.04  | arm64 |    6     | ami-06dbd7a3e3569babc | ami-06dbd7a3e3569babc |    False     |
|                |         |       |          |                       |                       |              |
|   ap-east-1    |  18.04  | amd64 |    6     | ami-0116f92341e381aa1 | ami-0116f92341e381aa1 |    False     |
|                |         |       |          |                       |                       |              |
|   ap-east-1    |  16.04  | arm64 |    9     | ami-044f509770640155b | ami-044f509770640155b |    False     |
|                |         |       |          |                       |                       |              |
|   ap-east-1    |  16.04  | amd64 |    9     | ami-038ff3475cbb62351 | ami-038ff3475cbb62351 |    False     |
|                |         |       |          |                       |                       |              |
|   af-south-1   |  20.04  | arm64 |    4     | ami-0154a536a0f270308 | ami-0154a536a0f270308 |    False     |
|                |         |       |          |                       |                       |              |
|   af-south-1   |  20.04  | amd64 |    4     | ami-030b8d2037063bab3 | ami-030b8d2037063bab3 |    False     |
|                |         |       |          |                       |                       |              |
|   af-south-1   |  18.04  | arm64 |    5     | ami-09ec75309b64050d9 | ami-09ec75309b64050d9 |    False     |
|                |         |       |          |                       |                       |              |
|   af-south-1   |  18.04  | amd64 |    5     | ami-0a5cd293a7343f047 | ami-0a5cd293a7343f047 |    False     |
|                |         |       |          |                       |                       |              |
|   af-south-1   |  16.04  | arm64 |    32    | ami-07d1ddc0a19021abb | ami-07d1ddc0a19021abb |    False     |
|                |         |       |          |                       |                       |              |
|   af-south-1   |  16.04  | amd64 |    32    | ami-063a9ea2ff5685f7f | ami-063a9ea2ff5685f7f |    False     |
|                |         |       |          |                       |                       |              |
+----------------+---------+-------+----------+-----------------------+-----------------------+--------------+

```

**Review by j5awry — APPROVED** [review]

**Comment by j5awry** [comment]
I'll let Phil take a look before merging, but LGTM!

File: aws_marketplace_ubuntu_scraper.py [hunk 1]
```diff
@@ -562,13 +525,49 @@ def scrape_marketplace(marketplace_url):
...
+@click.option(
+    '--scraper-data', type=click.File('r'), required=True,
+    show_default=True, default='quickstart_entries.json'
+)
+def report(scraper_data):
+    from prettytable import PrettyTable
+    t = PrettyTable()
+    t.field_names = ['Region', 'Release', 'Arch', 'Position', 'Quickstart AMI', 'Streams AMI', 'Needs update']
+    data = json.loads(scraper_data.read())
+    for region in data:
```
Comment (philroche) [inline aws_marketplace_ubuntu_scraper.py]: Can we sort the regions here please?

File: aws_marketplace_ubuntu_scraper.py [hunk 2]
```diff
@@ -562,13 +525,49 @@ def scrape_marketplace(marketplace_url):
             )
 
 
+def _streams_get_image(region, suite, arch):
+    cmd = ['/snap/bin/sstream-query-cpc', '--max=1',
```
Comment (philroche) [inline aws_marketplace_ubuntu_scraper.py]: We need to add the simplesteams snap as a dependency in the readme. Also we should be using 

```
/snap/bin/simplestreams.sstream-query
```

**Review by philroche — CHANGES_REQUESTED** [review]
Looks good. A few small change requests

File: aws_marketplace_ubuntu_scraper.py [hunk 3]
```diff
@@ -562,13 +525,49 @@ def scrape_marketplace(marketplace_url):
...
+           'root_store=ssd', '--output-format=%(id)s']
+    return subprocess.check_output(cmd, encoding='utf-8', universal_newlines=True)
+
+
+@click.command()
+@click.option(
+    '--scraper-data', type=click.File('r'), required=True,
+    show_default=True, default='quickstart_entries.json'
+)
+def report(scraper_data):
```
Comment (philroche) [inline aws_marketplace_ubuntu_scraper.py]: Can you change this to `quicklaunch-report` so we can have an equivalent report for the marketplace scraping?

**Review by philroche — CHANGES_REQUESTED** [review]

File: aws_marketplace_ubuntu_scraper.py (hunk 1 above)
Comment (toabctl) [reply aws_marketplace_ubuntu_scraper.py]: The entries in quickstart_entries.json are already sorted. But sorted anyway . So Fixed.

File: aws_marketplace_ubuntu_scraper.py (hunk 2 above)
Comment (toabctl) [reply aws_marketplace_ubuntu_scraper.py]: Fixed

File: aws_marketplace_ubuntu_scraper.py (hunk 3 above)
Comment (toabctl) [reply aws_marketplace_ubuntu_scraper.py]: Fixed

**Review by philroche — APPROVED** [review]
Thank you for those changes. +1

**Pull request merged**